scss/atoms/buttons/_buttons.scss

```diff
@@ -0,0 +1,89 @@
+/*
+* ==========================================================================
+* Buttons
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"mixins";
+
+.btn {
+	border: 1px solid transparent;
+	cursor: pointer;
+	letter-spacing: map-get($ui-btn, letter-spacing);
+	font-family: $ui-btn-font;
+	font-weight: map-get($ui-btn, font-weight);
+	margin: map-get($ui-btn, margin);
+	text-decoration: none;
+	text-transform: map-get($ui-btn, text-transform);
+	@include btn-colors($ui-btn-default);
+	@include btn-size($ui-btn-md);
+}
+
+.btn-group {
```

Comment:
Should probably do this differently so it doesn't rely on cascading markup

Maybe .btn-group--btn on the interior buttons?  My only concern there had been there could easily be too many .btn classes on the button elements